ArduPlane/wscript

```diff
@@ -1,40 +1,41 @@
-#!/usr/bin/env python
```

Comment:
You appear to have changed the line endings in this file, making review very difficult.  Please change the line endings back.